### DISCUSSION TITLE
Nextra: FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory

### DISCUSSION THREAD

**Discussion by ojasuno** [vercel/vercel#8333] [Help]
While running a [Nextra](https://nextra.vercel.app/) app locally, I am getting the following error

command
```
yarn build
```
error
```
FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
```

**Comment by amyegan** [comment]
Hi @ojasuno. Are you able to share your repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example)? I'm not sure exactly what's going wrong without digging in bit more.
